spark-connector/spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/SparkTransformConverter.java

```diff
@@ -122,6 +171,33 @@ public static org.apache.spark.sql.connector.expressions.Transform[] toSparkTran
                   sparkTransforms.add(
                       createSparkIdentityTransform(
                           String.join(ConnectorConstants.DOT, identityTransform.fieldName())));
```

Comment:
could you simple the code? String.join(ConnectorConstants.DOT, identityTransform.fieldName()

Reply:
done